images/installer/Dockerfile

```diff
@@ -22,7 +22,7 @@ USER root
 # configurations for the two images.
 RUN mkdir -p /usr/share/ansible/ && ln -s /opt/app-root/src /usr/share/ansible/openshift-ansible
 
-RUN INSTALL_PKGS="skopeo" && \
+RUN INSTALL_PKGS="skopeo openssl java-1.8.0-openjdk-headless python-passlib httpd-tools" && \
```

Comment:
For Python dependencies it's probably better to update `requirements.txt` instead of installing via RPM, for consistency with other python deps. The `assemble` script takes care of installing them at build time.

In `Dockerfile.rhel7` we install via RPM, and `python-passlib`, `httpd-tools` and `java` are already pulled in as dependencies.